app/controllers/users_controller.rb

```diff
@@ -33,7 +30,15 @@ def create
     password: params[:password]
     )
     if @user.save
+
       flash[:sucess] = "You've been added"
+      if User.last.email.include?("@theironyard.com")
```

Comment:
We could have multiple people sign up at once, and `User.last` may not be the user that just signed up (and may not even be the same `User.last` in the next line). Any reason you don't want to use `@user` here?